cli/npm/resolvers/common.rs

```diff
@@ -69,7 +69,6 @@ pub async fn cache_packages(
 
   let mut handles = Vec::with_capacity(packages.len());
   for package in packages {
-    assert_eq!(package.copy_index, 0); // the caller should not provide any of these
```

Comment:
This is not the case anymore, since a `copy_index > 1` might be the non-copy package now if the main `copy_index == 0` was not available for this system.